source/unsafe-messenger.js

```diff
@@ -73,6 +75,35 @@ export default function unsafeMessenger() {
 			editor.blur();
 		});
 	}
+
+	function monacoEditor(target) {
+		const editor = monaco.editor.getModels()[0];
+		let currentValue = editor.getValue();
+		sendBack(target, currentValue);
+
+		editor.onDidChangeContent((e) => {
+			// Check if the change was made by the user
+			if (e.isUserChange) {
+				// Check if the current value is different from the previous value
+				if (currentValue !== editor.getValue()) {
+					currentValue = editor.getValue();
+					throttle(50, sendBack)(target, currentValue);
```

Comment:
The function needs to be wrapped with throttle and then called later, it cannot be wrapped and called in the same place, it won't be throttled. 

It should wrap the function passed to onDidChangeContent or use a pre-throttled function inside it if required (like `throttledSend` seen in Ace's function)

Reply:
Fix ✅